src/index.js

```diff
@@ -81,7 +82,12 @@ class Wrapper {
 
     // Add the node to the fixture so it runs the connectedCallback().
     if (isRootNode) {
-      fixture.appendChild(node);
+      if (isStringNode) {
+          fixture.innerHTML = this.node.outerHTML;
```

Comment:
Alternatively could be:
```
fixture.innerHTML = node;
```

but we need `nodeFromHtml` to be called anyway as we need to retrieve `customElementDefinition`.